pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -696,71 +711,36 @@ private void setupCohortTagRoutes() {
 
     private void setupPubSub(@NonNull Config cfg, NotificationUtil notificationUtil) {
         String projectId = cfg.getString(GCP_PATH_PUBSUB_PROJECT_ID);
-        String subscriptionId = cfg.getString(GCP_PATH_PUBSUB_SUB);
         String dsmToDssSubscriptionId = cfg.getString(GCP_PATH_DSM_DSS_SUB);
         String dsmTasksSubscriptionId = cfg.getString(GCP_PATH_DSM_TASKS_SUB);
         String mercuryDsmSubscriptionId = cfg.getString(GCP_PATH_MERCURY_DSM_SUB);
         String antivirusDsmSubscriptionId = cfg.getString(GCP_PATH_ANTI_VIRUS_SUB);
 
-        logger.info("Setting up pubsub for {}/{}", projectId, subscriptionId);
```

Comment:
This was listening for covid-19 results, which we no longer deliver, so I'm removing the code.